File: ClientApp/src/models/Comment.ts
export class Comment{
  id: number;
  content: string;
  author: string;
  date: string;
  post_id: number;
  constructor(Content:string,Author: string, Date: string, Post_id:number) {

    this.content=Content;
    this.author=Author;
    this.date=Date;
    this.post_id=Post_id;
  }
}

File: Models/Comment.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Backend_Foto.Models
{
    public class Comment
    {
        public int ID { get; set; }
        public string Content { get; set; }
        public string Author { get; set; }
        public string Date { get; set; }
        public int Post_Id { get; set; }
    }
}
